components/sup/src/main.rs

```diff
@@ -189,7 +189,7 @@ fn sub_run(m: &ArgMatches, launcher: LauncherCli) -> Result<()> {
                     &msg.bldr_channel
                         .clone()
                         .map(ChannelIdent::from)
-                        .unwrap_or_default(),
+                        .expect("update_svc_load_from_input to always set to Some"),
```

Comment:
This part threw me.  Shouldn't it be `bldr_channel is always set to Some`?

Reply:
What I'm saying is that `update_svc_load_from_input` is the function responsible for ensuring we have a `Some(…)` here:

> I `expect` `update_svc_load_from_input to always set` the value `to Some`.

Maybe that's confusingly worded, but I did want to call out `update_svc_load_from_input` because this `expect` is predicated on its implementation.

If there's a wording you like better, I'm open to changing it.

Reply:
Ohhhh, now that you explain it that way, it makes total sense.  Maybe `update_svc_load_from_input should always set this value to Some`. ?  I dunno 😁